XNAControls/IControlINIAttributeParser.cs

```diff
@@ -1,13 +1,20 @@
-﻿using Rampastring.Tools;
+using Rampastring.Tools;
 
-namespace Rampastring.XNAUI.XNAControls
+namespace Rampastring.XNAUI.XNAControls;
+
+/// <summary>
+/// Provides an interface for an object that can parse
+/// and apply custom INI attributes for controls.
+/// </summary>
+public interface IControlINIAttributeParser
 {
     /// <summary>
-    /// Provides an interface for an object that can parse
-    /// and apply custom INI attributes for controls.
+    /// Attempts to parse given key's value and sets the parameter value for the given control.
     /// </summary>
-    public interface IControlINIAttributeParser
-    {
-        bool ParseAttributeFromINI(XNAControl control, IniFile iniFile, string key, string value);
-    }
-}
\ No newline at end of file
+    /// <param name="control">The control that the parsing happens for currently.</param>
+    /// <param name="iniFile">The INI file that is being read from.</param>
+    /// <param name="key">The key that is being read.</param>
+    /// <param name="value">The key's value.</param>
+    /// <returns>Whether the parsing was succesful.</returns>
+    public bool ParseINIAttribute(XNAControl control, IniFile iniFile, string key, string value);
```

Comment:
The `public` keyword here is unnecessary. Everything defined in interfaces is public by default.